### PR TITLE
Dockerfile to build with PyMOAB

### PR DESCRIPTION
**Pull request by kkiesling**
DO NOT MERGE YET

This is a temporary script that can be used to build PyNE using PyMOAB. There are references to repos that should not be used in the final version.

### CODE REVIEW

File: ubuntu_16.04-pymoab.dockerfile [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+  && cd DAGMC \
+  #&& git remote add lucas https://github.com/ljacobson64/DAGMC.git \
+  #&& git fetch lucas \
+  && git checkout -b moab_cmake_var origin/moab_cmake_var \
+  && cd .. \
+  && mkdir bld \
+  && cd bld \
+  && cmake ../DAGMC/ -DMOAB_ROOT=$HOME/opt/moab/ -DCMAKE_INSTALL_PREFIX=$HOME/opt/dagmc \
+  && make \
+  && make install
```
Comment (gonuke) [inline ubuntu_16.04-pymoab.dockerfile]: delete the `build` and `src` dirs to cleanup like we did with MOAB?

File: ubuntu_16.04-pymoab.dockerfile [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+  && cd dagmc \
+  && git clone https://github.com/ljacobson64/DAGMC.git \
+  && cd DAGMC \
+  #&& git remote add lucas https://github.com/ljacobson64/DAGMC.git \
+  #&& git fetch lucas \
+  && git checkout -b moab_cmake_var origin/moab_cmake_var \
+  && cd .. \
+  && mkdir bld \
+  && cd bld \
+  && cmake ../DAGMC/ -DMOAB_ROOT=$HOME/opt/moab/ -DCMAKE_INSTALL_PREFIX=$HOME/opt/dagmc \
```
Comment (kkiesling) [inline ubuntu_16.04-pymoab.dockerfile]: update this to `-DMOAB_DIR`

File: ubuntu_16.04-pymoab.dockerfile [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+# add DAGMC to path
+ENV LD_LIBRARY_PATH $HOME/opt/dagmc/lib:$LD_LIBRARY_PATH
+ENV LIBRARY_PATH $HOME/opt/dagmc/lib:$LIBRARY_PATH
+
+
+# Install PyNE
+RUN cd $HOME/opt \
+    && git clone https://github.com/CNERG/pyne.git \
+    && cd pyne \
+    && git checkout -b pymoab_cleanup origin/pymoab_cleanup \
```
Comment (kkiesling) [inline ubuntu_16.04-pymoab.dockerfile]: repo and branch will be updated post pymoab merge

**Comment by bam241** [comment]
I am thinking that we could tag all of them with the same name, to avoid having all the temporary one in memory, that could also allow users to build a big container with multiple of our softwares/dependencies

**Comment by gonuke** [comment]
I think this was superseded by #31 - maybe @kkiesling or @bam241 can confirm?

**Comment by gonuke** [comment]
This has been superseded by #35, merged today

**Comment by kkiesling** [comment]
This is no longer relevant then right? Can I close this PR?

**Comment by gonuke** [comment]
please do close it

**Pull request closed without merging**